packages/client/hmi-client/src/components/documents/Document.vue

```diff
@@ -81,6 +81,14 @@
 						label="Open PDF"
 						@click="openPDF"
 					/>
+					<Button
+						v-if="doi"
+						class="p-button-sm p-button-outlined"
+						@click="downloadPDF"
+						:icon="isEmpty(pdfLink) ? 'pi pi-spin pi-spinner' : 'pi pi-cloud-download'"
```

Comment:
This is nice, but you can use the [`:loading` property](https://primevue.org/button/#loading), will make it easier to read.